packages/legacy-client/src/client/actions/actions.test.ts

```diff
@@ -32,14 +32,13 @@ import {
   it,
   vi,
 } from "vitest";
-import {
-  type ActionError,
-  ActionExecutionMode,
-  type ActionExecutionOptions,
-  type ActionResponseFromOptions,
-  type Edits,
-  type Result,
-  ReturnEditsMode,
+import { ActionExecutionMode, ReturnEditsMode } from "../..";
+import type {
```

Comment:
This happened automatically on save, not sure if this is what we want

Reply:
🤷 unless you want to propose changes to dprint's config its fine by me